pages/nft/url/index.vue

```diff
@@ -89,6 +89,21 @@ import {
   API_POST_ARWEAVE_UPLOAD,
 } from '~/constant/api'
 
+const base64toBlob = (base64Data:string, contentType: string, sliceSize = 512) => {
```

Comment:
What does `sliceSize` mean here? Would 512 always work?

Reply:
I mainly refer to the Internet and see that many people use 512
https://stackoverflow.com/questions/16245767/creating-a-blob-from-a-base64-string-in-javascript
https://www.796t.com/post/MTF4YQ==.html